lib/auth_pts.c

```diff
@@ -400,7 +401,7 @@ static int ptload(const char *identifier, struct auth_state **state)
 
         syslog(LOG_DEBUG,
                "ptload(): fetched cache record (%s)" \
-               "(mark %ld, current %ld, limit %ld)", identifier,
+               "(mark " TIME_T_FMT ", current %" PRIu64 ", limit %" PRIu64 ")", identifier,
```

Comment:
I think all three of these should be TIME_T_FMT, not PRIu64? Note that at line 398/399 `now` is declared as `time_t`, and the printout of "current" and "limit" are calculated from `now`.

Reply:
Yeah, they do need to be TIME_T_FMT.  Build fails on linux/gcc/x86_64 with PRIu64's here.

Reply:
Right, got symbols-resolved var `now` to function `now()` which is `int64_t`. Will fix this on the next fixing iteration.